src/pages/sprikipedia/team/index.astro

```diff
@@ -14,9 +14,19 @@ const data = (
 <Categories name={data["team.categories.name"]} title={data["team.categories.title"]} >
     <p>{data["team.categories.about"]}</p>
```

Comment:
I appreciate your changes but I'm already preparing for spriki29 with reference to https://github.com/hackclub/sprig/blob/main/RESPONSIBILITIES.md :)